README.md

```diff
@@ -42,7 +42,7 @@ Next you may want to read:
 - [**Parameters**](https://github.com/microsoft/LightGBM/blob/master/docs/Parameters.rst) is an exhaustive list of customization you can make.
 - [**Distributed Learning**](https://github.com/microsoft/LightGBM/blob/master/docs/Parallel-Learning-Guide.rst) and [**GPU Learning**](https://github.com/microsoft/LightGBM/blob/master/docs/GPU-Tutorial.rst) can speed up computation.
 - [**Laurae++ interactive documentation**](https://sites.google.com/view/lauraepp/parameters) is a detailed guide for hyperparameters.
-- [**FLAML**](https://www.microsoft.com/en-us/research/project/fast-and-lightweight-automl-for-large-scale-data/articles/flaml-a-fast-and-lightweight-automl-library/) provides automated tuning for LightGBM ([code examples](https://github.com/microsoft/FLAML/blob/main/notebook/flaml_lightgbm.ipynb)).
+- [**FLAML**](https://www.microsoft.com/en-us/research/project/fast-and-lightweight-automl-for-large-scale-data/articles/flaml-a-fast-and-lightweight-automl-library/) provides automated tuning for LightGBM ([code examples](https://microsoft.github.io/FLAML/docs/Examples/AutoML-for-LightGBM)).
```

Comment:
```suggestion
- [**FLAML**](https://www.microsoft.com/en-us/research/project/fast-and-lightweight-automl-for-large-scale-data/articles/flaml-a-fast-and-lightweight-automl-library/) provides automated tuning for LightGBM ([code examples](https://microsoft.github.io/FLAML/docs/Examples/AutoML-for-LightGBM/)).
```

Can we add this trailing slash? I know it's a minor thing, but that will change the response from a 3xx to a 200, which might matter for the automated check this project runs on external links ([see here](https://github.com/microsoft/LightGBM/blob/90a71b9403e7facf52e2973ccdd6403a8071c898/.github/workflows/linkchecker.yml)).

```shell
curl -I https://microsoft.github.io/FLAML/docs/Examples/AutoML-for-LightGBM | head -1
#  HTTP/2 301

curl -I https://microsoft.github.io/FLAML/docs/Examples/AutoML-for-LightGBM/ | head -1
# HTTP/2 200
```